portfolio/src/main/webapp/index.html

```diff
@@ -143,24 +141,35 @@ <h2 id = "gallery" > Gallery </h2>
                 <img class = "slide" src="/images/group3.jpg">
             </div>
 
+            <div class = "mySlides">
+                <img class = "slide" src="/images/canada.jpg">
+            </div>
+
+            <div class = "mySlides">
+                <img class = "slide" src="/images/littlepecks2.jpg">
+            </div>
+
             <a class = "prev" onclick="plusSlides(-1)">❮</a>
             <a class = "next" onclick="plusSlides(1)">❯</a>
 
         </div>
        
       <h2 id = "contact" > Contact </h2>
+        <div class = "clearfix">
```

Comment:
I'm not sure what the name `clearfix` means in context, and I cannot find `.clearfix` in CSS.

Reply:
Thanks I learned something new. Found the css now, it was covered because it was added to the stylesheet in a previous PR